### PR TITLE
Add newly discovered info about how translator comments can fail.

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

**Comment by nedbat** [comment]
@lamagnifica @sarina OK?

**Comment by sarina** [comment]
:+1:

File: en_us/developers/source/internationalization/i18n.rst [hunk 1]
```diff
@@ -111,7 +111,24 @@ instead::
     message = _("Welcome!")
 
 "Translators" comments will work in these places too, so don't be shy about
-providing clarifying comments to the translators.
+providing clarifying comments to the translators. When writing translators
```
Comment (lamagnifica) [inline en_us/developers/source/internationalization/i18n.rst:114]: would it make sense to consistently icap the Translators in the phrase Translators comments ?

**Comment by lamagnifica** [comment]
:+1: please add a change log entry.
Also, just to raise awareness _for the future_, the team has developed style guidelines to make translation go more smoothly. Your addition has a consistent style with the rest of this file as it is now, so no worries for this PR, but going forward we are trying to minimize use of contractions, compound sentences (such as those with two clauses separated by a colon) and use of sentence fragments that end with a colon.

**Pull request merged**